libs/api/domains/social-insurance/src/lib/resolvers/medicalDocuments.resolver.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  CurrentUser,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+  type User,
+} from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+import { Audit } from '@island.is/nest/audit'
+import { FeatureFlagGuard } from '@island.is/nest/feature-flags'
+import { UseGuards } from '@nestjs/common'
+import { Query, Resolver } from '@nestjs/graphql'
+import { RehabilitationPlanModel } from '../models/medicalDocuments/rehabilitationPlan.model'
+import { SocialInsuranceService } from '../socialInsurance.service'
+
+@Resolver()
+@UseGuards(IdsUserGuard, ScopesGuard, FeatureFlagGuard)
+@Audit({ namespace: '@island.is/api/social-insurance' })
+@Scopes(ApiScope.internal)
+export class MedicalDocumentsResolver {
+  constructor(private readonly service: SocialInsuranceService) {}
+
+  @Query(() => RehabilitationPlanModel)
+  async siaGetRehabilitationPlan(@CurrentUser() user: User) {
+    return this.service.getRehabilitationPlan(user)
+  }
```

Comment:
The query name should reflect the object being fetched
https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions
```suggestion
  @Query(() => RehabilitationPlanModel, {
     name: 'socialInsuranceRehabilitationPlan'
  })
  async siaGetRehabilitationPlan(@CurrentUser() user: User) {
    return this.service.getRehabilitationPlan(user)
  }
```

Reply:
Fixed